docs/Decentraland Protocol for Explorers.md

```diff
@@ -0,0 +1,143 @@
+
+### Content layer
+
+The Decentraland Explorer and other applications render representations of the assets from the consensus layer. The content layer defines the mechanisms in which the content is:
+
+1. **Signed**, **validated** and **uploaded** to a node
+2. **Replicated** to other nodes and
+3. **Served** to the final users
+
+To do so, a [content-server](https://github.com/decentraland/catalyst) was created. It is a service that runs inside a Catalyst.
+
+> A _catalyst node_ is a bundle of services that self-contain a copy of Decentraland. The DAO [governs a curated list of trusted catalysts](https://governance.decentraland.org/?type=catalyst) that hold a working copy of Decentraland. It contains everything necessary to make the Explorer work.
+
+The content server stores and synchronizes _Entities_.
+
+An [Entity](https://github.com/decentraland/common-schemas/blob/be7213b40a2180a9a99035eb87e8a5d4b8438e7f/src/platform/entity.ts#L21-L37) is a signed data structure holding a list of content files, a deployment date and pointers. A pointer is a human readable "shortcut" to the entity i.e. the [`0,0`](https://peer.decentraland.org/content/entities/scenes?pointer=0,0) representation of a LAND. Every time an entity is deployed and accepted by the network, the pointers for that entity will now point to the newest entity for that set of pointers. That is the mechanism used to change the content of the land, wearables and emotes.
+
+Unlike LAND, wearables and emotes have [URN pointers](https://github.com/decentraland/urn-resolver/blob/b11aeb677e06e1a9e1d7994efa98a5f11867f854/test/urn.spec.ts#L138-L147). URN are used to reference any asset inside Decentraland, the technology was selected in pursue of a common identifier that enables [extensibility to other networks or remote assets](https://github.com/decentraland/urn-resolver/blob/b11aeb677e06e1a9e1d7994efa98a5f11867f854/test/urn.spec.ts#L269-L280) and [interoperability](https://github.com/common-metaverse/urn-namespaces) with other platforms.
+
+> TODO: Describe entities format
+
+> TODO: Describe entities content restrictions
+
+> TODO: Describe entities IPFS CIDs
+
+> TODO: Describe AuthChain
+
+#### Endpoints that are part of the protocol
+
+##### `GET /contents/:cid` Download an entity or content file
+
+Used to download the content. This endpoint is used by the content-server synchronization and by the explorer.
+
+##### `POST /entity` Upload a new entity
+
+The `POST /entity` must run validations to either accept or reject an entity. Depending on its type and querying the Consensus layer.
+
+This endpoint is used by the tooling to upload content.
+
+##### `POST /entities/active` Query active entities
+
+Used to discover the map around the user. This endpoint is used by the explorer.
+
+##### TODO: Get and Deploy profiles
```

Comment:
what should be added here? Isn't it the same as the endpoints above?

Reply:
Maybe, if we include lambdas then it isn't. It may be a good moment to also describe the endpoint in the bff